pkg/nfd-worker/nfd-worker.go

```diff
@@ -218,9 +219,11 @@ func (w *nfdWorker) startGrpcHealthServer(errChan chan<- error) error {
 // Run feature discovery.
 func (w *nfdWorker) runFeatureDiscovery() error {
 	discoveryStart := time.Now()
+	w.featureSourceErrors = 0
```

Comment:
I think this can be more meaningful as a variable name
```suggestion
	w.numberOffeatureSourceErrors = 0
```